<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frdWidget</class>
 <widget class="QWidget" name="frdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>865</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Features Reads Distribution</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1174</width>
        <height>839</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout2">
         <item>
          <widget class="QWidget" name="plotPooledLogWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>550</width>
             <height>400</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="plotPooledNormalWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>550</width>
             <height>400</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="plotLogWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>550</width>
             <height>400</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="plotNormalWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>400</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>550</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
